scripts/check_if_pr_is_low_risk.py

```diff
@@ -0,0 +1,317 @@
+                % line)
+            return [], {}
+        elif len(split) == 2:
+            diff_files.append((split[1], split[1]))
+        elif len(split) == 3:
+            diff_files.append((split[1], split[2]))
+    file_diffs = {}
+    for file_tuple in diff_files:
+        for filename in file_tuple:
+            if filename in file_diffs:
+                continue
+            if filename not in FILES_THAT_NEED_DIFFS:
+                continue
+            file_diff = common.run_cmd([
+                'git', 'diff', '-U0',
+                '{}/{}'.format(UPSTREAM_REMOTE, base_branch),
+                '--', filename,
+            ])
+            file_diff_split = file_diff.rstrip().split('\n')
+            i = 0
+            for line in file_diff_split:
+                i += 1
+                if line.startswith('@@'):
+                    break
+            if i > 6 or i == len(file_diff_split):
+                python_utils.PRINT(
+                    'Skipped too many lines when parsing "%s" diff'
+                    % filename)
+                return [], {}
+            file_diffs[filename] = file_diff_split[i:]
+    return diff_files, file_diffs
+
+
+def lookup_pr(owner, repo, pull_number):
+    """Lookup a PR using the GitHub API.
+
+    Args:
+        owner: str. Owner of the repository the PR is in.
+        repo: str. Repository the PR is in.
+        pull_number: str. PR number.
+
+    Returns:
+        dict. JSON object returned by the GitHub API v3. This is an
+        empty dictionary if the response code from the GitHub API is not
+        200.
+    """
+    request = python_utils.url_request(
+        GITHUB_API_PR_ENDPOINT % (owner, repo, pull_number),
+        None,
+        {'Accept': 'application/vnd.github.v3+json'})
+    response = python_utils.url_open(request)
+    if response.getcode() != 200:
+        return {}
+    pr = json.load(response)
+    response.close()
+    return pr
+
+
+def check_if_pr_is_translation_pr(pr, diff_files, unused_file_diffs):
+    """Check if a PR is low-risk by virtue of being a translation PR.
+
+    To be a low-risk translation PR, a PR must:
+
+    * Be opened from a branch on the oppia/oppia repository.
+    * Be opened from the branch `translatewiki-prs`.
+    * Only change JSON files in `assets/i18n`.
+    * Not change the names of any files.
+    * Be opened to the branch `develop`
+
+    Args:
+        pr: dict. JSON object of PR from GitHub API.
+        diff_files: list(tuple(str, str)). Changed files, each as a
+            tuple of (old name, new name).
+        unused_file_diffs: dict(str, list(str)). Map from file names to
+            the lines of that file's diff.
+
+    Returns:
+        str. An empty string if the PR is a translation PR and low-risk,
+        else a message explaining why the PR is not low-risk.
+    """
+    source_repo = pr['head']['repo']['full_name']
+    if source_repo != 'oppia/oppia':
+        return 'Source repo is not oppia/oppia'
```

Comment:
This is because we rely on branch protection rules to restrict who can push to translation and changelog branches. Those rules only apply on the `oppia/oppia` repo, so we have to ensure PRs are opened from branches on `oppia/oppia`.

While we want to enforce this for both of the current types of low-risk PRs, that may not be the case in the future. I'm trying to not make assumptions about what kinds of PRs we might want to skip CI checks on in the future, which is why I kept this check in the individual checker functions and not in `main`. I expect we will add more kinds of low-risk PRs in the future, which is why I wrote this code to be extensible.